src/main/java/Tuesday/Tuesday.java

```diff
@@ -0,0 +1,53 @@
+package Tuesday;
+
+import Tuesday.command.Command;
+import Tuesday.command.Parser;
+import Tuesday.task.Task;
+import Tuesday.util.Storage;
+import Tuesday.util.TuesdayException;
+import Tuesday.util.Ui;
+
+public class Tuesday {
+    // variable
+    private Storage storage;
+    private Task tasks;
+    private Ui ui;
+
+    /**
+     * Creates Tuesday.Tuesday.util.Tuesday.util.Ui, Tuesday.Tuesday.util.Tuesday.util.Storage & Tuesday.Tuesday.task.Tuesday.task.Task Object, and checks whether there exist a datafile
+     */
+    public Tuesday(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
```

Comment:
Have you considered the situation where the file or folder doesn't exist?

Reply:
Thanks for the help. I forgot to consider that.